multiqc/modules/biscuit/biscuit.py

```diff
@@ -0,0 +1,688 @@
+                ('SAligned', {'name':'Suboptimally Aligned Reads', 'color': '#f7a35c'}),
+                ('UAligned', {'name':'Unaligned Reads', 'color': '#000000'})
+            ]), {'id':'biscuit_mapping_summary',
+                 'title':'BISCUIT: Mapping Summary',
+                 'ylab':'Number of Reads',
+                 'cpswitch_c_active': True,
+                 'cpswitch_counts_label': '# Reads'
+            })
+        )
+
+        # mapping strand distribution
+        pd = dict([(k.replace('_strand_table.txt', ''),v) for k,v in self.mdata['mapq'].items() if k.endswith('_strand_table.txt')])
+        self.add_section(
+            name='Mapping Strand Distribution',
+            anchor='biscuit-strands',
+            description = "<p>This plot shows the distribution of strand of mapping and strand of bisulfite conversion.</p>",
+            plot = bargraph.plot(pd, OrderedDict([
+                ('++', {'name':'Waston-Aligned, Waston-Bisulfite Conversion', 'color': '#F53855'}),
+                ('+-', {'name':'Waston-Aligned, Crick-Bisulfite Conversion', 'color': '#E37B40'}),
+                ('-+', {'name':'Crick-Aligned, Waston-Bisulfite Conversion', 'color': '#46B29D'}),
+                ('--', {'name':'Crick-Aligned, Crick-Bisulfite Conversion', 'color': '#324D5C'})
+            ]), {'id':'biscuit_strands',
+                 'title':'BISCUIT: Mapping Strand Distribution',
+                 'ylab':'Number of Reads',
+                 'cpswitch_c_active': True,
+                 'cpswitch_counts_label': '# Reads'
+            })
+        )
+
+        # Mapping Quality, Insert Size
+        total = {}
+        for sid, dd in self.mdata['mapq'].items():
+            if not sid.endswith('_mapq_table.txt'):
+                continue
+            sid = sid.replace('_mapq_table.txt','')
+            total[sid] = sum([int(cnt) for _, cnt in dd.items() if _ != "unmapped"])
+
+        pd_mapping = {}
+        for sid, dd in self.mdata['mapq'].items():
+            if not sid.endswith('_mapq_table.txt'):
+                continue
+            sid = sid.replace('_mapq_table.txt','')
+            mapqcnts = []
+            for mapq in range(61):
+                if str(mapq) in dd:
+                    mapqcnts.append(float(dd[str(mapq)])/total[sid]*100)
+                else:
+                    mapqcnts.append(0)
+            pd_mapping[sid] = dict(zip(range(61), mapqcnts))
+
+        pd_isize = dict([(k.replace('_isize_score_table.txt', ''),v['I']) for k,v in self.mdata['mapq'].items() if k.endswith('_isize_score_table.txt')])
+
+        self.add_section(
+            name = 'Mapping Quality and Insert Size',
+            anchor = 'biscuit-mapq-isize',
+            description = "<p>This plot shows the distribution of Primary Mapping Quality.</p>",
+            plot = linegraph.plot([pd_mapping, pd_isize],
+                {'id':'biscuit_mapping', 'title': 'Mapping Information', 'data_labels': [
```

Comment:
Please prepend the title with the module name.

Reply:
Suggested additional config options:
```python
'ymin': 0,
'yCeiling': 100,
'yMinRange': 10,
'yLabelFormat': '{value}%',
'tt_label': '<strong>Q{point.x}:</strong> {point.y:.2f}% of reads'
```

Reply:
These are added accordingly.